### PR TITLE
add filter

### PR DESCRIPTION
**Pull request by MissMelisa**
Add filter to the main page through the buttons 
image

### CODE REVIEW

File: src/Components/foodItems/index.jsx [hunk 1]
```diff
@@ -10,8 +10,10 @@ const useStyles = makeStyles({
     margin: "30px",
   },
   itemName: {
-    fontSize: "25px",
+    fontSize: "1rem",
     fontWeight: "bolder",
+    fontFamily: "Open Sans,sans-serif",
```
Comment (falecci) [inline src/Components/foodItems/index.jsx:15]:
```suggestion
```

Let's just use the Material UI default font family so we keep consistency across our typography.

File: src/Pages/MainPage/index.jsx [hunk 2]
```diff
@@ -128,63 +137,84 @@ const useStyles = makeStyles({
...
+          </Button>
         ))}
       </div>
+
+      <div className={classes.containerCategory}>
+        {categories
+          .filter((category) => !filter || (filter && category.name === filter))
+          .map((category) => (
+            <div className={classes.category}>
+              <h1 className={classes.categoryTitle}>{category.name}</h1>
```
Comment (falecci) [inline src/Pages/MainPage/index.jsx:202]: Let's use `Typography` component here.

File: src/Pages/MainPage/index.jsx [hunk 3]
```diff
@@ -128,63 +137,84 @@ const useStyles = makeStyles({
...
   return (
     <div className={classes.page}>
       <img
         className={classes.logo}
         alt="logo"
-        src="Images/StakeHouseBurgers.jpg"
+        src="https://i.pinimg.com/originals/e2/98/11/e29811d3411c6696130a123c32727d9a.jpg"
       />
-      <h1> Stoke House Burgers</h1>
+      <h1 className={classes.restaurantName}> Stoke House Burgers</h1>
```
Comment (falecci) [inline src/Pages/MainPage/index.jsx:168]: Let's use `Typography` component here.

**Review by falecci — CHANGES_REQUESTED** [review]

File: src/Components/foodItems/index.jsx (hunk 1 above)
Comment (MissMelisa) [reply src/Components/foodItems/index.jsx:15]: ok

File: src/Pages/MainPage/index.jsx (hunk 3 above)
Comment (MissMelisa) [reply src/Pages/MainPage/index.jsx:168]: ok

File: src/Pages/MainPage/index.jsx (hunk 2 above)
Comment (MissMelisa) [reply src/Pages/MainPage/index.jsx:202]: ok

**Review by falecci — APPROVED** [review]

**Pull request merged**